examples/fluids/src/setuplibceed.c

```diff
@@ -510,7 +512,8 @@ PetscErrorCode SetupLibceed(Ceed ceed, CeedData ceed_data, DM dm, User user,
   // CEED Operator Apply
   // *****************************************************************************
   // -- Apply CEED Operator for the geometric data
-  CeedOperatorApply(ceed_data->op_setup_vol, ceed_data->x_coord, ceed_data->q_data,
+  CeedOperatorApply(ceed_data->op_setup_vol, ceed_data->x_coord,
+                    ceed_data->q_data,
```

Comment:
Every once in a while, `make style` results in awkward breaks like this one.

Reply:
Ouch! Thank you for the catch!